### PR TITLE
Invalid url for external content

### PR DESCRIPTION
**Pull request by jomaora**
**Detailed purpose of the PR**

A new property was added to show a warning icon when URL are invalid for external contents and a tooltip is shown on hover.
Delete button was also updated according to zpl://screen?sid=5e5d23b7c9fe3566633fddb3&pid=5cae10fc99bbed05850fba04

**Result and observation**

warning+tooltip

**Testing Strategy**

- Manual testing
- Unit testing

### CODE REVIEW

**Review by StefanoA1 — APPROVED** [review]

File: packages/@coorpacademy-components/src/molecule/drag-and-drop/upload-report.css [hunk 1]
```diff
@@ -81,45 +86,64 @@
...
+  min-width: 30px;
   position: relative;
 }
 
-.actionIcon:hover {
-  background: white;
-  box-shadow: 0 0 5px dark;
+.warningIcon {
+  border-radius: 50%;
+  background: #ff7043;
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/drag-and-drop/upload-report.css:118]:
```suggestion
  background: orangeAdd;
```

File: packages/@coorpacademy-components/src/molecule/drag-and-drop/upload-report.css [hunk 2]
```diff
@@ -81,45 +86,64 @@
...
+  color: white;
+  cursor: pointer;
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/drag-and-drop/upload-report.css:120]:
```suggestion
  cursor: help;
```

File: packages/@coorpacademy-components/src/molecule/drag-and-drop/upload-report.css [hunk 3]
```diff
@@ -81,45 +86,64 @@
...
   margin: auto;
 }
+
+.actionIcon > svg {
+  margin: 3px 5px 0 5px;
+}
+
 .trashIcon {
-  background: #f73f52;
+  color: #f73f52;
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/drag-and-drop/upload-report.css:140]:
```suggestion
  color: negative;
```

File: packages/@coorpacademy-components/src/molecule/drag-and-drop/upload-report.css [hunk 4]
```diff
@@ -81,45 +86,64 @@
...
 }
+
 .trashIcon:hover {
   color: #f73f52;
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/drag-and-drop/upload-report.css:144]:
```suggestion
  color: negative;
```

File: packages/@coorpacademy-components/src/molecule/drag-and-drop/upload-report.js [hunk 5]
```diff
@@ -61,13 +63,29 @@ export const UploadReport = ({
     </div>
   );
   const showMessage = ['success', 'error'].includes(state);
+
+  const warningMessageView = warningMessage ? (
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/drag-and-drop/upload-report.js:67]:
```suggestion
  const warningMessageView = !isEmpty(warningMessage) ? (
```

Comment (esa-coorp) [reply packages/@coorpacademy-components/src/molecule/drag-and-drop/upload-report.js:67]: or:

```suggestion
  const warningMessageView = <div className={style.showToolTip}>
      <span>{warningMessage}</span>
    </div>
```

**Review by esa-coorp — CHANGES_REQUESTED** [review]

**Review by esa-coorp — APPROVED** [review]

**Pull request merged**